polymath/extensions/item_ops.py

```
@@ -3,457 +3,473 @@
-        Qube: The object with swapped items.
-    """
+        from polymath.qube import Qube
 
-    new_values = self._values
-    len_shape = new_values.ndim
+        new_values = self._values
+        len_shape = new_values.ndim
 
-    for r in range(self._nrank):
-        new_values = np.rollaxis(new_values, -self._drank-1, len_shape)
+        for r in range(self._nrank):
```

Ruff (0.14.7)

Loop control variable `r` not used within loop body

Rename unused `r` to `_r`

(B007)